doc/nrf/gs_testing.rst

```diff
@@ -21,3 +21,26 @@ Connect with the following settings:
  * 1 stop bit
  * No parity
  * HW flow control: None
+
+.. _lte_connect:
+
+How to connect with LTE Link Monitor
+************************************
+
+To connect to nRF9160 based board devices, you can also use `LTE Link Monitor`_ which is implemented in `nRF Connect for Desktop`_.
+This application is used to establish LTE communication with the nRF9160 modem through AT commands.
+
+To connect to the nRF9160 based board device with LTE Link Monitor, perform the following steps:
+
+1. Launch the LTE Link Monitor app.
+
+   .. note::
+
+      Make sure that **Automatic requests** is enabled in LTE Link Monitor.
+
+#. Connect the nRF9160 based board device to the PC with a USB cable.
+#. Power on the nRF9160 based board device.
+#. Click **Select Device** and select the particular board device entry from the drop-down list in the LTE Link Monitor.
+#. Observe that the LTE Link monitor app starts AT communication with the modem of the nRF9160 based board and shows the status of the communication in the display terminal.
+
+   See the `AT Commands Reference Guide`_ for information on the various AT commands used in the communication.
```

Comment:
I don't think we need this here, because this topic is more about the UART output - it should stay in the AT client doc.

Reply:
Removed this.